src/road/traffic.py

```diff
@@ -66,6 +66,21 @@ def __init__(self, id, node: RoadSegmentNode):
         # Intersection
         self.wait_time = 0  # sec
 
+    @property
+    def speed(self):
+        """
+        Returns the speed of the vehicle in terms of tiles traveled
+        """
+        return self.vehicle_speed * tw
```

Comment:
`vehicle_speed` is assumed to be the actual distance traveled in the world grid per frame and that's how it will be referenced. I think the convention of saving `vehicle_speed` as `<multiplier> * tw` for now makes the most sense. I don't see an advantage of abstracting this calculation away from the actual value.